src/golang/lib/models/shared/list.go

```diff
@@ -0,0 +1,62 @@
+package shared
+
+import (
+	"database/sql/driver"
+	"sort"
+
+	"github.com/aqueducthq/aqueduct/lib/models/utils"
+	"github.com/dropbox/godropbox/errors"
+)
+
+var ErrReadOnlyFieldType = errors.New("The cannot serialized read-only field type.")
```

Comment:
We don't want to use this pattern for defining errors anymore. See [here](https://github.com/aqueducthq/aqueduct/blob/main/src/golang/lib/database/database.go#L15) for an example of how we should define this. Also we should use our `lib//errors` package instead of the dropbox package.

Reply:
Also typo in the error message